gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/equality/EqualityTest.java

```diff
@@ -57,10 +61,11 @@ public void testCollectionEqualityByGraphElementIds() throws Exception {
 
     FlinkAsciiGraphLoader loader = getTestGraphLoader();
 
-    CollectionEquality equality = new CollectionEquality(
-      new GraphHeadToEmptyString(),
-      new VertexToIdString(),
-      new EdgeToIdString(),
+    CollectionEquality<EPGMGraphHead, EPGMVertex, EPGMEdge, LogicalGraph, GraphCollection> equality =
+      new CollectionEquality<>(
```

Comment:
Indentation: arguments to CollectionEquality constructor should have more indentation.